### PR TITLE
DOC: fixing up the numpy.ma.cov function doco,

### LINKED ISSUE #6327: Bug in documentation for numpy.cov
Concerning the parameters, 
http://docs.scipy.org/doc/numpy/reference/generated/numpy.cov.html#numpy.cov
says about rowvar:

rowvar : int, optional

```
If rowvar is non-zero (default), then each row represents a variable, with observations in the columns. Otherwise, the relationship is transposed: each column represents a variable, while the rows contain observations.
```

But looking at the source-code, we have:
    if rowvar:
        N = X.shape[1]
        axis = 0
    else:
        N = X.shape[0]
        axis = 1

in other words, rowvar is boolean. if rowvar is True (default), then each row represents a variable and observations are in columns. If rowvar is False, it means that each column represents a variable, and observations are in rows.

The current text is kind of confusion, as one might think rowvar can specify a range or specific rows or something.

The documentataion (or rather the comment in source-code) should be changed accordingly. Note that interestingly, http://docs.scipy.org/doc/numpy/reference/generated/numpy.ma.cov.html has it right

### PR DESCRIPTION
**Pull request by pizzathief**
closes #6327

### CODE REVIEW

File: numpy/lib/function_base.py [hunk 1]
```diff
@@ -2089,7 +2089,7 @@ def _vectorize_call(self, func, args):
         return _res
 
 
-def cov(m, y=None, rowvar=1, bias=0, ddof=None, fweights=None, aweights=None):
+def cov(m, y=None, rowvar=True, bias=0, ddof=None, fweights=None, aweights=None):
```
Comment (charris) [inline numpy/lib/function_base.py]: Line now need to be broken to keep is < 80 characters long.

**Comment by charris** [comment]
Couple of style nits, otherwise LGTM.

**Comment by charris** [comment]
Also, lines in commit message <= 72 characters and `Closes #6327` in message body.

**Comment by larsmans** [comment]
`bias` is also a boolean disguising as an int.

**Comment by charris** [comment]
@pizzathief LGTM.  Could you squash the commits and rewrite the first commit message?

**Comment by charris** [comment]
@pizzathief The commit message summary is too long. All lines in the commit message should be <=72 characters. Maybe something like `MAINT: fix numpy.lib.cov signature and docstring.` Then add the details in the message body. I changed `DOC` because the function signature was also changed.

**Comment by charris** [comment]
@pizzathief I fixed up the commit message in #6857, so closing this. Thanks.

**Pull request closed without merging**